makefiles/pseudomodules.inc.mk

```diff
@@ -129,3 +129,6 @@ NO_PSEUDOMODULES += periph_common
 PSEUDOMODULES += auto_init_skald
 PSEUDOMODULES += skald_ibeacon
 PSEUDOMODULES += skald_eddystone
+
+
```

Comment:
Any reason for the duplicate newline here?

Reply:
I did it on purpose to separate the block where pseudomodules are defined and with the second block with the documentation.
I often only keep one newline between related things and use two to distinguish between blocks.

It is only personal style leaking here and I can change it.

Reply:
I think I slightly prefer a single line, but I don't mind having two newlines here if you think it makes things more readable.

Reply:
I update it then.